### PR TITLE
Feature/wnb 25/create initial db entities

### PR DESCRIPTION
**Pull request by ikushlianski**
- adapt project to use Sequelize
- add Users table migration
- add test users (Seeders feature in Sequelize)
- add instructions to Readme on how to use Sequelize with our project

Database connection is to be set up in https://wannabook.atlassian.net/browse/WNB-90

### CODE REVIEW

File: migrations/20190503203544-create-user.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+'use strict';
```
Comment (letutsenik) [inline migrations/20190503203544-create-user.js]: Why should we use 'use strict' mode?

**Review by letutsenik — COMMENTED** [review]
1. I think we should create a "db" folder and move all assets (ex. migrations, seeders) related to the database to this folder

File: migrations/20190503203544-create-user.js (hunk 1 above)
Comment (ikushlianski) [reply migrations/20190503203544-create-user.js]: This file was in the docs, but I guess we can remove this line of course

**Comment by ikushlianski** [comment]
I will try but this might require chages to standard cli commands, which assume that we exactly this folder structure

File: db/models/user.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      firstName: DataTypes.STRING,
+      lastName: DataTypes.STRING,
+      email: DataTypes.STRING,
+      password: DataTypes.STRING,
+      access_tokens: DataTypes.STRING,
+      refresh_token: DataTypes.TEXT,
```
Comment (letutsenik) [inline db/models/user.js:11]: Why don't we use camelCase for attributes values?

**Review by letutsenik — APPROVED** [review]

**Pull request merged**